From 72c692605737c0c7521d85a0ebf51a4e7b997461 Mon Sep 17 00:00:00 2001
From: Max Ihlenfeldt <max@igalia.com>
Date: Wed, 13 Dec 2023 15:21:01 +0000
Subject: [PATCH] Backport "IWYU for
 content/browser/generic_sensor/frame_sensor_provider_proxy.cc"

This patch fixes a build error due to a missing include. It will be
included in upstream's 122 release.

Upstream-Status: Backport [https://crrev.com/c/5119742]
Signed-off-by: Max Ihlenfeldt <max@igalia.com>
---
 content/browser/generic_sensor/frame_sensor_provider_proxy.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/content/browser/generic_sensor/frame_sensor_provider_proxy.cc b/content/browser/generic_sensor/frame_sensor_provider_proxy.cc
index e24c144..dfb417a 100644
--- a/content/browser/generic_sensor/frame_sensor_provider_proxy.cc
+++ b/content/browser/generic_sensor/frame_sensor_provider_proxy.cc
@@ -4,6 +4,8 @@

 #include "content/browser/generic_sensor/frame_sensor_provider_proxy.h"

+#include <vector>
+
 #include "base/notreached.h"
 #include "content/browser/renderer_host/render_frame_host_impl.h"
 #include "content/public/browser/browser_context.h"
